src/libs/print_settings.c

```diff
@@ -1495,7 +1570,7 @@ void *legacy_params(dt_lib_module_t *self, const void *const old_params, const s
     }
 
     // in theory pprofile can't be srgb or adobergb, but checking for them won't hurt
-    if(*pprofile == '\0' || !g_strcmp0(pprofile, "none"))
+    if(*pprofile == '\0' || !g_strcmp0(pprofile, "server color managed"))
```

Comment:
If you are reusing the string here you should have it as a global `static const char*`. Besides, you are adding it as a translated string so I am not sure this will ever work when not using an English GUI.
Another point: you are looking at legacy params here. How should those ever contain the new string you just introduced? Shouldn't you keep checking for the old `none`?

Reply:
You're plain right, I should have used _("none") here.